### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by rahulneal**

### CODE REVIEW

**Comment by rahulneal** [comment]
hey i created P0 card elements please review.

File: component_styles/card.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+.card__container{
+    background-color: white;
```
Comment (shashikantmaurya0007) [inline component_styles/card.css:13]: create variable for color if you are using it multiple times

**Review by shashikantmaurya0007 — APPROVED** [review]
i have added few comment you can work on it .rest seems good to me

File: component_styles/card.css (hunk 1 above)
Comment (rahulneal) [reply component_styles/card.css:13]: ok

**Pull request merged**